.env.example

```diff
@@ -1,5 +1,5 @@
 # Which type of ethereum testnet to run locally (pow|pos) 
-ETH_TESTNET_TYPE="pow"
+ETH_TESTNET_TYPE="pos"
```

Comment:
I think I'd prefer pow for the example file. I usually run pow if I don't have to run pos